Sources/MapboxNavigation/BannerContainerView.swift

```diff
@@ -0,0 +1,288 @@
+import UIKit
+
+// :nodoc:
+@objc(MBBannerContainerView)
```

Comment:
For now I've marked newly added APIs as undocumented, will do that after performing more tests.

Reply:
One you do make these methods documented, remember to mention them (and overall support for animating the banners) in the changelog.